tools/cellprofiler/image_math.xml

```diff
@@ -0,0 +1,301 @@
+    </outputs>
+    <tests>
+        <test>
+            <param name="input_pipeline" value="common_image_math.cppipe"/>
+            <param name="operation" value="add"/>
+            <param name="raise_the_power_of_the_result_by" value="1.0"/>
+            <param name="multiply_the_result_by" value="1.0"/>
+            <param name="add_to_result" value="0.0"/>
+            <param name="set_values_less_than_0_equal_to_0" value="Yes"/>
+            <param name="set_values_greater_than_1_equal_to_1" value="Yes"/>
+            <param name="ignore_the_image_masks" value="No"/>
+            <param name="name_output_image" value="ImageAfterMath"/>
+            <param name="image_or_measurement_first" value="Image"/>
+            <param name="select_the_first_image" value="DNA"/>
+            <param name="multiply_the_first_image_by" value="1.0"/>
+            <param name="image_or_measurement_second" value="Image"/>
+            <param name="select_the_second_image" value="CellImage"/>
+            <param name="multiply_the_second_image_by" value="2.0"/>
+            <output name="output_pipeline" ftype="txt" file="image_math_1.cppipe" lines_diff="0"/>
+        </test>
+        <test>
+            <param name="input_pipeline" value="common_image_math.cppipe"/>
+            <param name="operation" value="subtract"/>
+            <param name="raise_the_power_of_the_result_by" value="1.0"/>
+            <param name="multiply_the_result_by" value="1.0"/>
+            <param name="add_to_result" value="0.0"/>
+            <param name="set_values_less_than_0_equal_to_0" value="Yes"/>
+            <param name="set_values_greater_than_1_equal_to_1" value="No"/>
+            <param name="ignore_the_image_masks" value="No"/>
+            <param name="name_output_image" value="ImageAfterMath"/>
+            <param name="image_or_measurement_first" value="Image"/>
+            <param name="select_the_first_image" value="DNA"/>
+            <param name="multiply_the_first_image_by" value="1.0"/>
+            <param name="image_or_measurement_second" value="Measurement"/>
+            <param name="multiply_the_second_image_by" value="5.0"/>
+            <param name="category_second" value="FileName"/>
+            <param name="measurement_second" value="DNA"/>
+            <output name="output_pipeline" ftype="txt" file="image_math_2.cppipe" lines_diff="0"/>
+        </test>
+        <test>
+            <param name="input_pipeline" value="common_image_math.cppipe"/>
+            <param name="operation" value="not"/>
+            <param name="name_output_image" value="ImageAfterMath"/>
+            <param name="select_the_first_image" value="DNA"/>
+            <param name="ignore_the_image_masks" value="No"/>
+            <output name="output_pipeline" ftype="txt" file="image_math_3.cppipe" lines_diff="0"/>
+        </test>
+    </tests>
+    <help>
+        <![CDATA[
+
+            .. class:: infomark
```

Comment:
I don't think you can use the indentation here. Have you look with planmo serve how this looks like?

Reply:
looks good :)